cognee-mcp/src/server.py

```
@@ -75,14 +74,15 @@ async def cognify_task(file_path: str) -> None:
             logger.info(f"Starting cognify for: {file_path}")
             try:
-                await cognee.add(file_path, nodeset="developer_rules")
+                await cognee.add(file_path, node_set=["developer_rules"])
                 model = KnowledgeGraph
                 if graph_model_file and graph_model_name:
                     model = load_class(graph_model_file, graph_model_name)
                 await cognee.cognify(graph_model=model)
                 logger.info(f"Cognify finished for: {file_path}")
             except Exception as e:
                 logger.error(f"Cognify failed for {file_path}: {str(e)}")
+                raise ValueError(f"Failed to cognify: {str(e)}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)